packages/buckets/main.tf

```diff
@@ -1,3 +1,14 @@
+resource "google_storage_bucket" "loki_storage_bucket" {
+  name     = "${var.gcp_project_id}-loki-storage"
+  location = var.gcp_region
+
+  public_access_prevention    = "enforced"
+  storage_class               = "STANDARD"
```

Comment:
Would you add a lifecycle policy?

Reply:
Yes, we want to add this — probably deletion after bucket objects being 192h old.